FetchMigration/python/orchestrator.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import logging
+import os
+import subprocess
+
+import migration_monitor
+import pre_migration
+from migration_monitor_params import MigrationMonitorParams
+from pre_migration_params import PreMigrationParams
+
+
+__DP_EXECUTABLE_SUFFIX = "/bin/data-prepper"
+__PIPELINE_OUTPUT_FILE_SUFFIX = "/pipelines/pipeline.yaml"
+
+
+def run(dp_base_path: str, dp_config_file: str, dp_endpoint: str):
+    dp_exec_path = dp_base_path + __DP_EXECUTABLE_SUFFIX
+    output_file = dp_base_path + __PIPELINE_OUTPUT_FILE_SUFFIX
```

Comment:
what will this output contain?  Will it be a report, logs?  It's fair for this component to leave that to DataPrepper if it's just stdout or labeled as output, but I'm still curious (so that as a reviewer, I can think about how the output needs to be shared/secured).

Reply:
`output_file` is the path to which `pre_migration.py` will write a modified Data Prepper pipeline YAML file after comparing indices between the source and target clusters. This pipeline YAML file ensures that only indices that are safe to migrate are created on the target cluster (and eventually migrated). The Data Prepper executable uses this `output_file` as its pipeline configuration.